src/waypoint.jsx

```diff
@@ -32,6 +45,20 @@ function debugLog() {
   console.log(arguments); // eslint-disable-line no-console
 }
 
+function parseOffsetAsPixels(str) {
```

Comment:
On second thought, maybe they should be. I don't think computing these over and over affect performance too much, but it's an easy change.